src/sentry/db/router.py

```diff
@@ -64,6 +67,16 @@ class SiloRouter:
     For now they are put in control silo.
     """
 
+    historical_silo_assignments = {
```

Comment:
I get the feeling this will be infrequently used but it might be worth pointing people here from the error raised in `allow_migrate` if you think it will happen often

Reply:
The errors that occur when this fallback is missing happen far away from our code unfortunately. When the pagerduty model wasn't resolved it ended up being a missing table much later in the migration history.